source/precalculus/source/08-TE/01.ptx

```diff
@@ -12,6 +12,1187 @@
+            <ol marker= "A." cols="1">
+              <li> <m>d=\sqrt{1-\sin\alpha\sin\beta-\cos\alpha\cos\beta}</m> </li>
+              <li> <m>d=\sqrt{2-2\sin\alpha\sin\beta-2\cos\alpha\cos\beta}</m></li>
+              <li>  <m>d=\sqrt{2-2\cos\alpha\cos\beta-2\sin\alpha\sin\beta}</m> </li>
+              <li> <m>d=\sqrt{2-\sin\alpha\sin\beta-\cos\alpha\cos\beta}</m> </li></ol></p>
+        </statement>
+        <hint>
+          <p>
+            You may need to rearrange some of your terms before applying the Pythagorean Identity.
+          </p>
+        </hint>
+        <answer>
+          <p>
+            B or C
+          </p>
+        </answer>
+      </task>
+
+      <task>
+        <statement>
+          <p>
+            Using the same steps as in parts (c-e), use the distance formula to find the distance between points <m>A</m> and <m>B</m>.
+            <ol marker= "A." cols="1">
+              <li> <m>d=\sqrt{1-2\cos\left(\alpha-\beta\right)}</m> </li>
+              <li>  <m>d=\sqrt{2-2\cos\left(\alpha-\beta\right)}</m></li>
+              <li>  <m>d=\sqrt{1-\cos\left(\alpha-\beta\right)}</m> </li>
+              <li> <m>d=\sqrt{-2\cos\left(\alpha-\beta\right)+2}</m> </li></ol></p>
+        </statement>
+        <hint>
+          <p>
+            Point <m>A</m> has coordinates <m>(\cos (\alpha - \beta), \sin (\alpha - \beta))</m> and point <m>B</m> has coordinates <m>(1,0)</m>.
+          </p>
+        </hint>
+        <answer>
+          <p>
+            B or D
+          </p>
+        </answer>
+      </task>
+
+      <task>
+        <statement>
+          <p>
+            From part (b), we know that the length of <m>PQ</m> is equal to the length of <m>AB</m>. Therefore, <me>\sqrt{2-2\cos\alpha\cos\beta-2\sin\alpha\sin\beta}=\sqrt{2-2\cos\left(\alpha-\beta\right)}</me>. Square both sides of this equation and solve for <m>\cos\left(\alpha-\beta\right)</m>.
+          </p>
+        </statement>
+        <answer>
+          <p>
+            <m>\cos\left(\alpha-\beta\right)=\cos\alpha\cos\beta+\sin\alpha\sin\beta</m>
+          </p>
+        </answer>
+      </task>
+    </activity>
+
+    <remark>
+      In <xref ref="derive-difference-formula-for-cosine"/>, we derived the difference formula for cosine. The development of the sum/difference formulas are similar for sine and cosine and same for <m>\tan(\alpha-\beta)</m> compared to <m>\tan(\alpha+\beta)</m>. 
+    </remark>
+
+    <definition xml:id="def-sum-and-difference-formulas">
+      <statement>
+        <p>
+          The <term>Sum and Difference Formulas</term> in trigonometry are used to find the value of the trigonometric functions at specific angles where it is easier to express the angle as the sum or difference of unique angles such as <m> 0 </m> <degree/>, <m>30</m><degree/>, <m>45</m><degree/>, <m>60</m><degree/>, <m>90</m><degree/>, and <m>180</m><degree/>.
```

Comment:
```suggestion
          The <term>Sum and Difference Formulas</term> in trigonometry are used to find the value of the trigonometric functions at specific angles where it is easier to express the angle as the sum or difference of unique angles such as <m> 0^\circ </m>, <m>30^\circ</m>, <m>45^\circ</m>, <m>60^\circ</m>, <m>90^\circ</m>, and <m>180^\circ</m>.
```

Reply:
Note that `<degree/>` [is intended]([Math as Text](https://pretextbook.org/doc/guide/html/topic-mathematics-practices.html#topic-mathematics-practices-9) 
https://pretextbook.org/doc/guide/html/topic-mathematics-practices.html#topic-mathematics-practices-9) for non-mathematical uses.  Since you are typesetting the numbers as math, just use `\circ` in the math.  This prevents line breaks between the number and the degree symbol.